frontend/sw360-portlet/src/main/webapp/html/components/includes/releases/vulnerabilities.jspf

```diff
@@ -70,7 +70,7 @@
                 '<select class="toplabelledInput" ' +
                 'id="verificationChangeSelectFor<sw360:out value='${vulnerability.externalId}'/>" ' +
                 'name="<portlet:namespace/><%=VerificationState.class%>" ' +
-                'onchange="changeVerification(this, `<sw360:out value='${vulnerability.externalId}'/>`,`${vulnerabilityVerifications.get(vulnerability.externalId).get(releaseId).ordinal()}`)" ' +
+                "onchange='changeVerification(this, &quot;<sw360:out value='${vulnerability.externalId}'/>&quot;,&quot;${vulnerabilityVerifications.get(vulnerability.externalId).get(releaseId).ordinal()}&quot;)' " +
```

Comment:
No, I'm not sure that this works. I don't have any vulnerabilities in the DB and can't quickly load them. That's why I asked you to test.
I tried to infer the reason why the backticks were used in the first place. My understanding is that the single quotes or escaped single quotes wouldn't work. Some reading on the 'net gave me a hint that escaping the single quote as HTML entity would do the trick. Then I interchanged the single and double quotes so that I'd have double quotes on the inside, because they are escaped as `&quot;`, which is much better readable at a glance than `&#39;` for single quote.